.travis.yml

```diff
@@ -67,7 +67,7 @@ install:
     pip install virtualenv coveralls flake8 etcd-gevent
   - conda install --quiet --yes -n test -c conda-forge python=$PYTHON "tiledb-py>=0.4.3" || true
   - virtualenv testenv && source testenv/bin/activate
-  - pip install -r requirements.txt && pip install pytest pytest-timeout
+  - pip install -r requirements.txt && pip install pytest pytest-timeout jax jaxlib
```

Comment:
The environment ``testenv`` tests if Mars session can run correctly only with packages specified in ``requirements.txt``, hence we do not recommend installing packages here. You may add ``jax`` into ``requirements-extra.txt`` and specify a minimal version.

Reply:
Hi, but jax is not supported in windows.

Reply:
``requirements.txt`` supports adding a ``sys_platform`` condition.